dvc/repo/experiments/executor/base.py

```diff
@@ -450,9 +452,32 @@ def _repro_args(cls, dvc):
             kwargs = {}
         return args, kwargs
 
+    @staticmethod
+    def _auto_push(git_remote: str, dvc: "Repo", scm: "Git"):
+        from dvc.repo.experiments.push import push
+
+        if git_remote == scm.root_dir:
+            logger.warning(
+                f"{DVC_EXP_AUTO_PUSH} {git_remote} is the running "
+                "repository auto push will not work"
+            )
+            return
+
+        branch = scm.get_ref(EXEC_BRANCH, follow=False)
+        branch_name = ExpRefInfo.from_ref(branch).name
+        push(
+            dvc,
+            git_remote,
+            branch_name,
+            push_cache=True,
+            run_cache=True,
+        )
+        logger.info({"pushed": branch_name})
```

Comment:
I think this is probably unneeded (and too verbose). We already log when a push is completed in `experiments.push` at the `debug` level.